tests/data/reference_graphs/const_sparsity/sr_small_model.dot

```diff
@@ -50,41 +50,6 @@ strict digraph  {
 "48 SmallModel/NNCFConv2d[conv_output]/conv2d" [id=48, scope="SmallModel/NNCFConv2d[conv_output]", type=conv2d];
 "49 SmallModel/__mul__" [id=49, scope=SmallModel, type=__mul__];
 "50 SmallModel/add" [id=50, scope=SmallModel, type=add];
-"51 SmallModel/NNCFConv2d[conv_cubic1]/conv2d" [id=51, scope="SmallModel/NNCFConv2d[conv_cubic1]", type=conv2d];
```

Comment:
strange, that there's the deletion of nodes only

Reply:
That's because the reference graph prior to this change actually contained duplicate nodes of the majority of the model - which is fixed by this PR